.wellcome_project

```diff
@@ -72,6 +72,63 @@ catalogue_pipeline:
       name: Production
     - id: miro_merging_test
       name: Miro merging test
+  image_repositories:
+    - id: id_minter
+      namespace: uk.ac.wellcome
```

Comment:
Is it worth using an alias/anchor for the repeated account ID and namespace keys?

Reply:
It's worth doing something to reduce repetition (as previously mentioned by @alexwlchan). I'm not a massive fan of references in YAML. I think the deploy tool should provide a mechanism to specify (and override) defaults.

Reply:
yeah that sounds like a better idea - also not a huge fan of YAML references